testem.js

```diff
@@ -1,9 +1,17 @@
-{
+/*jshint node:true*/
+
+var launchInCI = function() {
+  if (!process.env.TRAVIS || process.env.TRAVIS_PULL_REQUEST) {
```

Comment:
That’s new as of [2.4.0](https://github.com/ember-cli/ember-new-output/commit/66d108894c9612ffe0e29ec36c749bedd21e5aeb). Useful!